tools/install.sh

```diff
@@ -4,7 +4,7 @@
 
 REPO=${REPO:-novaframework/nova}
 REMOTE=${REMOTE:-https://github.com/${REPO}.git}
-BRANCH=${BRANCH:-master}
+BRANCH=${BRANCH:-lfe-templating}
```

Comment:
This needs to be master if we are merging it :)

Reply:
Ah yes. Should  change it back.